.flake8

```diff
@@ -33,3 +16,16 @@ exclude =
     ctypes,
     pydispatch,
     testsuite,
+
+per-file-ignores =
+    ./actinia_core/resources/common/app.py: E402, E501
```

Comment:
I find it helpful to have a comment somewhere explaining the PEP8 codes. For actinia-module-plugin e.g. I added them on top: https://github.com/mundialis/actinia-module-plugin/blob/main/.flake8#L3